### PR TITLE
Deprecate legacy decode instructions pipeline

### PR DESCRIPTION
**Pull request by desmond-hui**
> [!IMPORTANT]  
> This will not be merged until **Monday 12/9 630am PT**

- Remove calls to legacy pipeline
- Modify realtime queries to match existing but using `streamline.complete_decoded_instructions_3`
- Modify `silver.decoded_instructions` to use streamline 2.0 upstream data after cutoff data `2024-12-09`
  - Temporarily extend lookback window
- Modify backfill macro to call streamline 2.0 endpoint

Streamline models run on DEV
`dbt run -s streamline__decode_instructions_3_realtime -t dev`
```
17:30:44  1 of 1 OK created sql table model streamline.decode_instructions_3_realtime .... [SUCCESS 1 in 528.25s]
```

`dbt run -s streamline__decode_instructions_3_pyth_realtime -t dev`
```
17:32:40  1 of 1 OK created sql view model streamline.decode_instructions_3_pyth_realtime  [SUCCESS 1 in 3.57s]
```

Silver model on DEV
`dbt run -s silver__decoded_instructions -t dev-2xl`
```
16:56:07  1 of 1 OK created sql incremental model silver.decoded_instructions ............ [SUCCESS 26068385 in 111.25s]
```

### CODE REVIEW

**Comment by tarikceric** [comment]
Should the `dbt_run_decode_instructions_orchestrator` be updated to remove `complete_decoded_instructions_2` as well?

File: macros/helpers/decoded_instructions_backfill_helpers.sql
Comment (tarikceric) [inline macros/helpers/decoded_instructions_backfill_helpers.sql:1]: do any of the other helper functions need to be updated to use `xxx_3` models?

**Comment by desmond-hui** [comment]
> Should the `dbt_run_decode_instructions_orchestrator` be updated to remove `complete_decoded_instructions_2` as well?

Decided to leave it for now, shouldn't take any resources really as it should get no new data

File: macros/helpers/decoded_instructions_backfill_helpers.sql
Comment (desmond-hui) [reply macros/helpers/decoded_instructions_backfill_helpers.sql:1]: Good call, I will switch it so that it defaults to look at `xxx_3` models

**Review by tarikceric — APPROVED** [review]

**Pull request merged**